ui/app/assets/javascripts/fusor_ui/application_no_tl.js

```diff
@@ -0,0 +1,620 @@
+//= require jquery
+//= require i18n
```

Comment:
Same as foreman's [application.js](https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/application.js#L2-L3) without the turbolinks files and renamed application_no_tl.js